### PR TITLE
Ts/update stratify beaker context

### PR DESCRIPTION
**Pull request by Tom-Szendrey**
# Whats changed?
- Moving stratify context to the `mira_model_edit` context where a stratify procedure exists (https://github.com/DARPA-ASKEM/beaker-kernel/pull/130)
- Reset thoughts on asking a question
- append thoughts instead of replacing
- append llm_responses as well (this is when a user asks for documentation for eg)
- Update stratify default Q's

# Note: 
below the code screenshots have a bunch of comments above the stratify code. This is a beaker change in another branch that will not be keeping the comments.

# Pictures 
image

image

### CODE REVIEW

**Review by shawnyama — DISMISSED** [review]

**Review by shawnyama — APPROVED** [review]
Nice. Part of me wonders if the `thought-output` component could be within the `jupyter-input` since the llmThoughts are generated there anyways. Maybe there is a benefit to having it in the drilldown that idk about.

**Comment by Tom-Szendrey** [comment]
> Nice. Part of me wonders if the `thought-output` component could be within the `jupyter-input` since the llmThoughts are generated there anyways. Maybe there is a benefit to having it in the drilldown that idk about.

I do see what you mean. I think this way is slightly easier to debug, and more explicit. Both are valid options

File: packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-thought-output.vue [hunk 1]
```diff
@@ -11,18 +11,35 @@
...
+});
+
+// Set model, modelNodeOptions
+watch(
+	() => props.llmThoughts,
+	async () => {
+		console.log(props.llmThoughts);
+	},
+	{ immediate: true }
+);
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-thought-output.vue:37]: Is this still needed?

File: packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-thought-output.vue [hunk 2]
```diff
@@ -11,18 +11,35 @@
...
 const props = defineProps<{
-	llmThought?: any;
+	llmThoughts: any[];
 }>();
 const showThoughts = ref(false);
-const thought = computed(() => props?.llmThought?.content?.thought ?? '');
+const thought = computed(() => {
+	let aString = '';
+	props.llmThoughts.forEach((ele) => {
+		const llmResponse = ele.content?.thought ?? ele.content?.text ?? '';
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-thought-output.vue:24]: Could Beaker makes sure this is sent to only one property? or those are completely different things?

File: testing/Stratify-LLM-Tests.md
Comment (YohannParis) [inline testing/Stratify-LLM-Tests.md:1]: Nice first pass, I will give it a go later on 🎉

**Review by YohannParis — APPROVED** [review]

File: packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-thought-output.vue (hunk 1 above)
Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-thought-output.vue:37]: Ha! Thank you my mistake

File: testing/Stratify-LLM-Tests.md
Comment (Tom-Szendrey) [reply testing/Stratify-LLM-Tests.md:1]: I wanted to spew thoughts and examples down while it was on my mind.
If its hard to follow let me know!

File: packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-thought-output.vue (hunk 2 above)
Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-thought-output.vue:24]: This is coming from https://github.com/jataware/beaker-kernel not https://github.com/DARPA-ASKEM/beaker-kernel
This is because one is an llm_thought object and the other is an llm_response object
I dont think its worth digging into this too much
